src/utilities/credentials/credentials.ts

```diff
@@ -253,12 +253,16 @@ export async function checkCredentialsStatus(
   credentials: SporranCredential[],
 ): Promise<void> {
   for (const credential of credentials) {
-    if (credential.status !== 'attested') {
+    if (!['pending', 'attested'].includes(credential.status)) {
       continue;
     }
     try {
       if (await isAttestationRevoked(credential.credential.rootHash)) {
         await saveCredential({ ...credential, status: 'revoked' });
+        return;
+      }
+      if (credential.status === 'pending') {
```

Comment:
I was surprised by the logic here initially, so I think a comment like this will make it clear
```suggestion
      // isAttestationRevoked did not throw, so it is on chain, but was pending before
      if (credential.status === 'pending') {
```